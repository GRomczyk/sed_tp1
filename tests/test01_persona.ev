00:01:00:0 in_port1 [-1.0,1]
00:02:00:0 info 1
