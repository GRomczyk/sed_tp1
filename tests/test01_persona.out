00:01:00:000:0 out_port1 [0.5, 0.2]
00:02:00:000:0 info_out1      0.21016
